src/connectors/mailjet.js

```diff
@@ -0,0 +1,102 @@
+import path from 'path';
+import axios from 'axios';
+import { isEmpty } from 'lodash';
+
+import { render } from '../services/renderer';
+
+const apiKey =
+  process.env.MAILJET_API_KEY && !isEmpty(process.env.MAILJET_API_KEY)
```

Comment:
not sure this variable are injected anywhere for api-auth. Can you have a check ?

Reply:
what do you mean? The provisioning update will follow, do you want to update the provisioning before merging this request?